app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -833,7 +833,7 @@ def build_toolbar_hide_button(id)
         return true if @record.host && @record.host.vmm_product.downcase == "workstation"
       when "miq_template_refresh"
         return true if @record && !@record.ext_management_system && !(@record.host && @record.host.vmm_product.downcase == "workstation")
-      when "miq_template_scan"
+      when "miq_template_scan", "image_scan"
```

Comment:
@h-kataria how's this? https://github.com/agrare/manageiq/commit/11b4cc33cd9536307f338c2e30997998c817a461 Using the same logic as the vm_scan/instance_scan.